### PR TITLE
Feature/containerize

### PR DESCRIPTION
**Pull request by blegros-exte**
Added steps to run pipeline on Github actions + Created steps for building docker image and then pushing it to docker hub.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      - name: Setup pip
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.6'
+#          cache: pip
+
+      - name: Setup js dependencies
+        run: yarn install
+
+      - name: Install pybind11
```
Comment (benj-moreau) [inline .github/workflows/build.yaml:29]: this step can be removed

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+RUN yarn run build
+
+FROM base as semantic-bot
+
+WORKDIR /usr/src/app
+
+# Copy source and built files
+COPY --chown=1000:1000 requirements.txt ./
+
+RUN pip install pybind11==2.2.4
```
Comment (benj-moreau) [inline docker/Dockerfile:52]: same here

Comment (blegros-exte) [reply docker/Dockerfile]: Fixed !

File: .github/workflows/build.yaml (hunk 1 above)
Comment (blegros-exte) [reply .github/workflows/build.yaml]: Also fixed

**Pull request merged**